src/core/get.ts

```diff
@@ -65,7 +81,13 @@ export const memoizeGet = (fn: Get) => {
     return value;
   };
 
+  Object.assign(memoizedFn, { displayName: fn.name });
```

Comment:
Is `displayName` used elsewhere? Not sure if this should actually be `name` instead (the JS property on functions, vs displayName being a React component concept)

Reply:
Whoops, this was a holdover from when I was debugging something. I'll delete it.